CONTRIBUTING.md

```diff
@@ -139,6 +145,24 @@ Make sure to signoff your commits before submitting a pull request.
 
 https://docs.pi-hole.net/guides/github/how-to-signoff/
 
+When developing locally, the following commands are useful to run regularly to check unit tests and linting.
+
+| Command  | Description                                        | Is called in the CI? |
+| make unit-test | Runs unit tests only. Good to run often when developing locally. `make all` will also run this. | yes                  |
+| make check-linter | Checks linter issues only. Good to run often when developing locally. `make all` will also run this. | yes                  |
```

Comment:
nit: 

now that this section is preceded by "When developing locally", the "Good to run often when developing locally" part of the description is redundant.

Reply:
https://github.com/ossf/scorecard/pull/3762/commits/1dcba3cdd8fde42f1129cd1da7c4d1957ea26ebd